components/browser/errorpages/src/main/assets/error_pages.html

```diff
@@ -8,32 +8,32 @@
     <head>
         <meta charset="UTF-8">
         <meta name="viewport" content="width=device-width; user-scalable=false;" />
-        <title>%pageTitle%</title>
-        <style>%css%</style>
+        <link rel="stylesheet" type="text/css" href="error_style.css">
     </head>
 
     <body id="errorPage" dir="auto">
         <!-- PAGE CONTAINER (for styling purposes only) -->
         <div id="errorPageContainer">
 
+            <!-- Error Image -->
+            <iframe id="errorImage" src="" frameborder="0"></iframe>
+
             <!-- Error Title -->
             <div id="errorTitle">
-                <h1 id="et_dnsNotFound" class="errorTitleText">%messageShort%</h1>
+                <h1 id="errorTitleText"/>
```

Comment:
nit: I think its not valid to have these tags be self closing (`h1`, `div`, `button`). Instead they should be empty: 
```suggestion
                <h1 id="errorTitleText"></h1>
```